ClangSharp.PInvokeGenerator.Test/StructDeclarationTest.cs

```diff
@@ -91,6 +91,58 @@ public partial struct MyNestedStruct
             await ValidateGeneratedBindings(inputContents, expectedOutputContents);
         }
 
+        [Fact]
+        public async Task SkipNoDefinitionTest()
+        {
+            var inputContents = "typedef struct MyStruct MyStruct;";
```

Comment:
This is the "opaque handle" case. Generally this will get consumed as a `MyStruct*` type elsewhere in the header file...

Reply:
It might be useful to change the "excludeFunctions" option to be "excludeName" instead, which will exclude any named declaration with a matching name so users can skip over unnecessary structs/enums as well.